src/util.ts

```diff
@@ -304,6 +304,14 @@ export async function screenshot(client: Client): Promise<string> {
   return screenshot.data
 }
 
+export async function pdf(client: Client): Promise<string> {
+  const {Page} = client
+
+  const pdf = await Page.printToPDF()
```

Comment:
Will it be hard to allow user pass [options](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-printToPDF) here?  In particular `landscape` may be useful